src/js/msp/MSPHelper.js

```diff
@@ -558,8 +563,12 @@ MspHelper.prototype.process_data = function(dataHandler) {
 
                     if (semver.eq(FC.CONFIG.apiVersion, '1.10.0')) {
                         // drop two unused servo configurations due to MSP rx buffer to small)
-                        while (FC.SERVO_CONFIG.length > 8) {
-                            FC.SERVO_CONFIG.pop();
+                        while (true) {
```

Comment:
This seems to be harder to read in the new form - why was this changed?

Reply:
Sonarcloud found it was incompliant. Had to rewrite this way as advised.

Reply:
Not sure SonarCloud is right here - I can't see any problem with the original code.

Reply:
Me neither - but it functions the same (the old way was not compliant).

Reply:
Please change this bach then please - using an infinite loop for no reason is confusing.